source/timezones.rst

```diff
@@ -0,0 +1,39 @@
+Timezones
+=========
+
+In order to get timezones working on a MariaDB/MySQL instance, you will have to initialize Timezones data and grant GLPI database user read ACL on their table.
+
+Non windows users
+-----------------
+
+On most systems, you'll have to initialize timezones data from your system's timezones:
+
+.. code-block:: bash
+
+   mysql_tzinfo_to_sql /usr/share/zoneinfo | mysql -u root -p mysql
```

Comment:
this can be disturbing, i suggest the following change
```suggestion
   mysql_tzinfo_to_sql /usr/share/zoneinfo | mysql -p -u root mysql
```

Reply:
As U wish, I've just copy/pasted the command I ran :p